### PR TITLE
#163478764 Implement Users should be able to comment on articles

### PR DESCRIPTION
**Pull request by fire-cracker**
#### What does this PR do?
> * Enables users to comment on an article and view comments history of an article
> 
 #### Description of Task to be completed?
> * Added comment component for comment on article
> * Added comment history component to show comments history of an article
> *
> 
 #### How should this be manually tested?
> * Use git to pull from the branch `feature/163478764/Article-Comment`
> * run `npm install` to install any new dependencies
> * run `npm dev -server` to start a server
> * Login using any user
> *  Select an article
> * Scroll down to the comment section of the article to Create a new comment or view the comment history to of the article.
> 
>

### CODE REVIEW

**Comment by jsamchineme** [comment]
Hi @oyedejipeace.

When a user tries to comment without being logged in, this is what they get. The screen shot below.

image

I think it would be nice if the toast message is just one concise message.

**Comment by jsamchineme** [comment]
@oyedejipeace 
### On the home page. This is what we have now on staging
image



### On your feature branch we have this. 
image

### Could you please address the issue.

**Comment by jsamchineme** [comment]
> @oyedejipeace
> 
> ### On the home page. This is what we have now on staging
> image
> 
> ### On your feature branch we have this.
> image
> 
> ### Could you please address the issue.

Apologies @oyedejipeace. I discovered that the environment variable pointing to the api, set on the testing server was wrong. 
There is no issue here. It's displaying as expected

File: src/components/comment/commentEntries.jsx [hunk 1]
```diff
@@ -1,16 +1,28 @@
 import React from "react";
 import PropTypes from 'prop-types';
+import { format } from 'date-fns';
 import './style.scss';
-import LikeNumberGroup from '../../components/LikeNumberGroup';
-import HorizontalLine from '../../components/HorizontalLine';
+import LikeNumberGroup from '../LikeNumberGroup';
```
Comment (ifiokeyo) [inline src/components/comment/commentEntries.jsx]: @oyedejipeace I gave feedback on this specifically to you and @jsamchineme. Why are you finding it difficult to adopt naming conventions ? Please make the required change.

File: src/components/comment/index.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+import TextAreaInput from '../TextAreaInput';
```
Comment (ifiokeyo) [inline src/components/comment/index.jsx]: @oyedejipeace If this import is from a directory, please refer to above comment.

**Review by ifiokeyo — CHANGES_REQUESTED** [review]

File: src/components/comment/commentEntries.jsx (hunk 1 above)
Comment (fire-cracker) [reply src/components/comment/commentEntries.jsx]: I am sorry if it seems like I ignored the feedback. That wasn't a deliberate action. I guess I didn't fully understand what was expected of me. I thought each of us was supposed to rename the files we named to follow the team's adopted naming conventions. I didn't realise I was supposed to do it for every file including the ones named by another team member. I
I have made all the necessary changes and going forward I will make sure to seek clarity when given an instruction.

**Pull request merged**